src/azure-cli/azure/cli/command_modules/mysql/tests/latest/test_mysql_scenario.py

```diff
@@ -126,6 +129,7 @@ def test_mysql_flexible_server_gtid_reset(self, resource_group):
     @ResourceGroupPreparer(location=DEFAULT_LOCATION)
     @KeyVaultPreparer(name_prefix='rdbmsvault', parameter_name='vault_name', location=DEFAULT_PAIRED_LOCATION, additional_params='--enable-purge-protection true --retention-days 90')
     @KeyVaultPreparer(name_prefix='rdbmsvault', parameter_name='backup_vault_name', location=DEFAULT_LOCATION, additional_params='--enable-purge-protection true --retention-days 90')
+    @unittest.skip('(AuthenticationError) The refresh token has expired or is invalid due to sign-in frequency checks by conditional access.')
     def test_mysql_flexible_server_byok_mgmt(self, resource_group, vault_name, backup_vault_name):
```

Comment:
Can we retry for auth error?

Reply:
I have retried and re-running az login or  az account get-access-token do not resolve this for me: https://github.com/Azure/azure-cli/issues/16209#issuecomment-787593844